qa/smoke-test-client/src/test/java/org/elasticsearch/smoketest/SmokeTestClientIT.java

```diff
@@ -27,10 +28,19 @@
 import static org.hamcrest.Matchers.greaterThan;
 
 public class SmokeTestClientIT extends ESSmokeClientTestCase {
+
+    // needed to avoid the test suite from failing for having no tests
+    // TODO: remove when Netty 4.1.5 is upgraded to Netty 4.1.6 including https://github.com/netty/netty/pull/5778
+    public void testSoThatTestsDoNotFail() {
+
+    }
+
     /**
      * Check that we are connected to a cluster named "elasticsearch".
      */
     public void testSimpleClient() {
+        // TODO: remove when Netty 4.1.5 is upgraded to Netty 4.1.6 including https://github.com/netty/netty/pull/5778
+        assumeFalse("JDK is JDK 9", Constants.JRE_IS_MINIMUM_JAVA9);
```

Comment:
@nik9000 I looked into it because we are accumulating a little bit of debt here to be paid when 4.1.6 is released. Here's the problem. This test uses a randomized client (sometimes it uses the mock transport plugin, sometimes Netty 4, and sometimes the default (Netty 4, this should be fixed so the randomization selects Netty 3). So, the first thing that would have to be done is expose which transport type the client is using. Then, we have to tear open the netty-common jar and read the jar manifest to parse out the `netty-common.version`. This seems like way more trouble than it's worth to me?